### PR TITLE
Design improvements

### PR DESCRIPTION
**Pull request by ishandongol**
- improved the UI
- changed the fonts and font-size
- customized pages
- improved the design of embed modal
- improved design of navigation bar
- switched to SASS

### CODE REVIEW

**Review by cliftonmcintosh — APPROVED** [review]
Looks good. I've put tested it out with our dockerized setup, and things seem to work. I'm curious about the `class=""` for the h1s. What does declaring an empty class do?

File: wazimap_np/static/css/components/_pageheader.scss [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#page-header{
+    @extend %card-shadow;
+    border-radius: 0;
+    // position: relative;
+    // height: $top-nav-height;
+    // #page-header-title{
+        // @include centerer;
+    // }
```
Comment (cliftonmcintosh) [inline wazimap_np/static/css/components/_pageheader.scss:8]: were these left in intentionally?

**Comment by cliftonmcintosh** [comment]
One thing to note is that it would be good to make sure that our changes do not make it hard for us to bring in updates from the wazimap framework project. 

If some of these changes could be useful for other organizations that use wazimap, we might consider making a pull request to wazimap itself in order to help with future compatibility.

**Comment by cliftonmcintosh** [comment]
@ishandongol 

Thank you. Your changes have been deployed at https://beta.nepalmap.org

**Comment by cliftonmcintosh** [comment]
Here are screenshots from the old pre-federal site and the updated look on the beta site.

**Before the updates on pre-federal site: comparing two districts**
pre-federal-comparison

**After the updates on federal site: comparing two districts**
updated-comparison

**Comment by ishandongol** [comment]
Yes, was planning to increase the height of top bar.

**Comment by ishandongol** [comment]
Do you mean the use of sass or design?

**Pull request merged**